src/pages/organize/[orgId]/campaigns/[campId]/surveys/[surveyId]/submissions.tsx

```diff
@@ -45,17 +52,37 @@ const SubmissionsPage: PageWithLayout<SubmissionsPageProps> = ({
     (env) =>
       new SurveySubmissionsModel(env, parseInt(orgId), parseInt(surveyId))
   );
+  const theme = useTheme();
+  const isMobile = useMediaQuery(theme.breakpoints.down('md'));
 
   return (
     <>
       <Head>
         <title>{model.getData().data?.title}</title>
       </Head>
-      <ZUIFuture future={subsModel.getSubmissions()}>
-        {(data) => {
-          return <SurveySubmissionsList submissions={data} />;
-        }}
-      </ZUIFuture>
+      <Grid container direction={isMobile ? 'column' : 'row'}>
```

Comment:
You can add the `spacing={2}` property here to add margin/spacing between grid items in this container. Use that instead of having margin-left on the alert.